_non-cosmos/xrpl/assetlist.json

```diff
@@ -38,6 +38,76 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/xrp.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/xrpl/images/xrp.svg"
       }
+    },
+    {
+      "description": "Sologenic is disrupting the asset trading industry: Tokenized Securities, Crypto Assets & NFTs.",
+      "denom_units": [
+        {
+          "denom": "xrpl11278ecf9e",
+          "exponent": 0,
+          "aliases": [
+            "xrpl11278ecf9e"
+          ]
+        },
+        {
+          "denom": "solo",
+          "exponent": 15
+        }
+      ],
+      "base": "xrpl11278ecf9e",
+      "name": "Solo",
+      "display": "solo",
+      "symbol": "SOLO",
+      "type_asset": "unknown",
+      "coingecko_id": "sologenic",
```

Comment:
The coingecko id is "solo"
{"id":"solo-coin","symbol":"solo","name":"Sologenic"}

Reply:
Fixed